scripts/setup.py

```diff
@@ -10,33 +10,65 @@
 
 import util
 
-def create_makefile_template_defines(source_root, proof_root, litani):
+SRCDIR_TEXT = """
+# Absolute path to the root of the source tree.
```

Comment:
🍎 = It looks like you're actually setting this to a *relative* path below––relative to the proof directory

Reply:
The result after applying `$(abspath ...)` is an absolute path.  Computing the path from a path relative to PROOF_ROOT is what was already there.

Reply:
Not sure, but I think Mark meant that the variable will ultimately store absolute path. And that's true because we call `abspath` below (within `make`).